WcaOnRails/app/views/competitions_mailer/notify_users_of_results_presence.html.erb

```diff
@@ -0,0 +1,11 @@
+<p>
+  The results of <%= link_to @competition.name, competition_url(@competition) %> you've competed recently are now available.
+</p>
+
+<p>
+  Check out all your results <%= link_to "here", competition_results_by_person_url(@competition, anchor: @user.wca_id) %>.
+</p>
+
+<p>
+  Regards, <%= users_to_sentence @competition.managers %>.
```

Comment:
@SAuroux, any thoughts on what the closing part of this email should say? How about something like:

```
Regards,

The WCA Results Team on behalf of <%= users_to_sentence @competition.managers %>.
```